client/src/components/SponsorsGuests/GuestSection.tsx

```diff
@@ -0,0 +1,84 @@
+// components/GuestSection.tsx
+import { Montserrat } from "next/font/google";
+import Image from "next/image";
+
+const montserrat = Montserrat({
+  subsets: ["latin"],
+  weight: ["400", "600", "700"],
+});
+
+// Type definition for a guest item
+type Guest = {
+  name: string;
+  description: string;
+  link: string;
+  image?: string; // optional photo URL
+};
+
+interface GuestSectionProps {
+  guests: Guest[]; // list of guests passed from parent component
+}
+
+// GuestSection component renders the special guests section
+const GuestSection = ({ guests }: GuestSectionProps) => {
+  return (
+    <div
+      className="mt-12 w-full px-8 py-16"
+      style={{ backgroundColor: "#54595F" }}
+    >
+      <h2
+        className={`text-center text-3xl font-bold text-black ${montserrat.className}`}
+        style={{ lineHeight: "1.15em" }}
+      >
+        Special Guests
+      </h2>
+
+      <div className="mx-auto mt-12 grid max-w-6xl grid-cols-1 gap-8 md:grid-cols-2 lg:grid-cols-4">
+        {guests.map((guest, idx) => (
+          <div key={idx} className="text-center">
+            {/* Guest image box */}
+            <div
+              className="relative mx-auto mb-4 flex items-center justify-center overflow-hidden bg-white shadow-md"
+              style={{ width: "200px", height: "200px" }}
```

Comment:
You can use custom values with tailwind inside `className`. No need to use `style`.

i.e.
`className="w-[200px] h-[200px]`